osbs/tekton.py

```diff
@@ -425,6 +419,15 @@ def get_info(self, wait=False):
         r = self.os.get(self.pipeline_run_url)
         return r.json()
 
+    def is_succeeded(self):
```

Comment:
has_succeeded seems better to me

Reply:
Looking at how this is actually used (e.g. `if pipeline_run.is_succeeded():`), IMO even `success` or `run_successful` would be better:
- `if pipeline_run.success():`
- `if pipeline_run.run_successful():`